README.md

```diff
@@ -32,6 +32,14 @@ pip install .
 
 Add ``-e`` to install in [develop mode](http://pip.readthedocs.org/en/latest/reference/pip_install.html#editable-installs).
 
+``pvlib-python`` comes with Python bindings for NREL's solar position C code,
+however you must manually download the C code directly from [NREL](http://www.nrel.gov/midc/spa/).
+To use the NREL SPA algorithm,
+
+1. Download the SPA files from [NREL](http://www.nrel.gov/midc/spa/)
+2. Copy the SPA files into ``pvlib/spa_c_files``
+3. From this ``pvlib-python`` directory, run ``pip install .``
```

Comment:
Well @Calama-Consulting, after seeing this implementation do you want to stick to your proposed solution in #9? 

For the 0.1 release I'm equally ok with accepting this PR (with minor revisions) or removing the spa wrapper entirely. If we remove the spa wrappers for 0.1, then I would support somebody (not me) creating a separate spa wrapper package and reimplementing its usage in pvlib in a future release. 

Maybe we should take the conversation back over to that issue and tag some of the other users and contributors to ask for their input?